spec/presenters/vendor_api/single_application_presenter_spec.rb

```diff
@@ -1,26 +1,39 @@
 require 'rails_helper'
 
+# To avoid this test becoming too large, only use this spec to test complex
+# logic in the presenter. For anything that is passed straight from the database
+# to the API, make sure that spec/system/vendor_api/vendor_receives_application_spec.rb is updated.
 RSpec.describe VendorApi::SingleApplicationPresenter do
-  let(:application_form) do
-    create(:completed_application_form) do |form|
-      form.first_nationality = 'British'
-      form.second_nationality = 'American'
+  describe 'attributes.candidate.nationality' do
+    it 'returns nationality in the correct format' do
+      application_form = create(:completed_application_form, :with_completed_references, first_nationality: 'British', second_nationality: 'American')
+      application_choice = create(:application_choice, status: 'awaiting_provider_decision', application_form: application_form)
+
+      response = VendorApi::SingleApplicationPresenter.new(application_choice).as_json
```

Comment:
This method is probably misnamed as it doesn't return JSON, it returns a hash — I was a bit confused by the way we call `to_json` next